roomserver/api/perform.go

```diff
@@ -105,8 +105,6 @@ type PerformInviteRequest struct {
 }
 
 type PerformInviteResponse struct {
-	// If non-nil, the invite request failed. Contains more information why it failed.
-	Error *PerformError
```

Comment:
Why was this removed? The lack of a dedicated type means we can't return appropriate error codes to the client.

Reply:
I did wonder about the API shape of that. I can put something back in.